src/utilities/domains.ts

```diff
@@ -40,3 +41,29 @@ export const getAbsoluteDomainPath = (
   const fixedPath = `${rootDomain}.${domain}`;
   return fixedPath;
 };
+
+// ID must be the given ID from CoinGecko API.
+// You can get this value from a token's page under "API ID"
+// https://www.coingecko.com/en/coins/zero-tech
+export const tokenAddressToFriendlyName: NetworksToAddresses = {
```

Comment:
Could we move this into `src/actions/getPaymentTokenInfo.ts` since it's the only place we use it?